docs/src/pages/components/text/react.mdx

```diff
@@ -82,26 +68,21 @@ To override styling on all Text, you can set the Amplify CSS variables or use th
 
 (Revise this example)
 
+<Example>
+  <GlobalStylingExample />
+  <ExampleCode>
 ```css
 /* styles.css */
 .primary-styled-variable-text.amplify-text {
```

Comment:
`.primary-styled-variable-text` makes this an example of local styling, this should be just `.amplify-text` or `[data-amplify-theme]` for the rule.

Reply:
Done!